node/tessel-export.js

```diff
@@ -671,27 +685,27 @@ Tessel.SPI = function(params, port) {
    *  with a max clock divisor of 255, slowest clock is 368Hz unless we switch from 48MHz xtal to 32KHz xtal
    */
   // default is 2MHz
-  params.clockSpeed = params.clockSpeed ? params.clockSpeed : 2e6;
+  this.clockSpeed = params.clockSpeed ? params.clockSpeed : 2e6;
 
   // if speed is slower than 93750 then we need a clock divisor
-  if (params.clockSpeed > 24e6 || params.clockSpeed < 368) {
+  if (this.clockSpeed > 24e6 || this.clockSpeed < 368) {
     throw new Error('SPI Clock needs to be between 24e6 and 368Hz.');
   }
 
-  this.clockReg = Math.floor(48e6 / (2 * params.clockSpeed) - 1);
+  this._clockReg = Math.floor(48e6 / (2 * this.clockSpeed) - 1);
 
   // find the smallest clock divider such that clockReg is <=255
-  if (this.clockReg > 255) {
+  if (this._clockReg > 255) {
     // find the clock divider, make sure its at least 1
-    this._clockDiv = Math.floor(48e6 / (params.clockSpeed * (2 * 255 + 2))) || 1;
+    this._clockDiv = Math.floor(48e6 / (this.clockSpeed * (2 * 255 + 2))) || 1;
 
     // if the speed is still too low, set the clock divider to max and set baud accordingly
     if (this._clockDiv > 255) {
-      this.clockReg = Math.floor(this.clockReg / 255) || 1;
+      this._clockReg = Math.floor(this._clockReg / 255) || 1;
       this._clockDiv = 255;
     } else {
       // if we can set a clock divider <255, max out clockReg
-      this.clockReg = 255;
+      this._clockReg = 255;
```

Comment:
I wanted it to stay consistent with the underscore naming for internal variables.